### PR TITLE
Modify the syntax specification of markdown

### PR DESCRIPTION
**Pull request by aimuz**
Fixed the following issues：

- Title level error
- Indent error
- Link error pointing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +38,14 @@ git submodule update --init --recursive --depth 1
...
-```
+```bash
 make container-image
 make container-serve
 ```
 
 If you see errors, it probably means that the hugo container did not have enough computing resources available. To solve it, increase the amount of allowed CPU and memory usage for Docker on your machine ([MacOSX](https://docs.docker.com/docker-for-mac/#resources) and [Windows](https://docs.docker.com/docker-for-windows/#resources)).
 
-Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+Open up your browser to <http://localhost:1313> to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
```
Comment (tengqm) [inline README.md:48]: Why adding the `<` and `>` characters? It will confuse the markdown parser which may believe it is a HTML tag!

Comment (aimuz) [reply README.md:48]: https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

This is the syntax of markdown，For naked URL

Comment (tengqm) [reply README.md:48]: Okay ... since these README files are only for github view. I'm fine with this change.

**Pull request merged**